Entities/ACRocket.h

```diff
@@ -38,6 +38,7 @@ class Leg;
 class ACRocket:
     public ACraft
 {
+	friend struct EntityLuaBindings;
```

Comment:
Do these need EntityLuaBindings as a friend? I assume if they didn't need LuaMan then no private stuff was being used before, and since you haven't changed any bindings, you shouldn't need it now??
Or is it just for future proofing?

Reply:
Need it to properly bind some of the private enums in the Actor classes.
Previously they weren't bound and if you made any changes to the enums you'd need to edit the values in the bindings as well.
I originally considered to make those enums public but this is cleaner.

![image](https://user-images.githubusercontent.com/50018358/127450599-071e0667-2ecd-4abe-b3ff-843bb82358b1.png)

![image](https://user-images.githubusercontent.com/50018358/127450617-809326cf-e9fa-43e3-9e46-0547d6265d26.png)
etc etc